lib/elixir/lib/enum.ex

```diff
@@ -3889,13 +3889,20 @@ defmodule Enum do
   Zips corresponding elements from two enumerables into a list
   of tuples.
 
+  Because a list of two-element tuples with atoms as the initial
+  tuple element is a keyword list (`Keyword`), zipping a first list
```

Comment:
```suggestion
  Because a list of two-element tuples with atoms as the first
  tuple element is a keyword list (`Keyword`), zipping a first list
```